ci/test.yml

```diff
@@ -319,3 +319,29 @@ storage test:
       junit: tests/junit.xml
     expire_in: 1 week
     when: always
+
+# Hardware
+hardware legacy device test:
+  stage: test
+  extends: .legacy_job
+  only:
+    - schedules  # nightly build
+    - /^legacy\//
+    - /^release\/[0-9]{4}-[0-9]{2}$/
```

Comment:
`/^release\// `

(we had branches like `release-2019-10-t1` in the past)

Reply:
Good point. Done and force pushed.